classes/basecontroller.php

```diff
@@ -43,11 +43,19 @@ public function __construct(\assign $assignment) {
     }
 
     protected function getheader($title) {
+        $settings = \assign_submission_comparativejudgement::getpluginsettings($this->assignment);
```

Comment:
Can we make this a setting that defaults to off?

Reply:
My guess is most people will want this so would default to on be possible?